.github/workflows/python-tests.yml

```diff
@@ -38,9 +38,12 @@ jobs:
           - os: ubuntu-latest
             python-version: 3.9
             toxenv: py39-test-pytest61
+          - os: ubuntu-latest
+            python-version: '3.10'
+            toxenv: py310-test-pytest71
           - os: macos-latest
-            python-version: 3.8
-            toxenv: py38-test-pytestdev
+            python-version: '3.10'
```

Comment:
Do we want Python 3.11 here?

Reply:
Sure, but it's enough to have it once the release is out

Reply:
🤞